openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -196,6 +197,9 @@ def award_program_certificates(self, username):
             try:
                 award_program_certificate(credentials_client, username, program_uuid)
                 LOGGER.info('Awarded certificate for program %s to user %s', program_uuid, username)
+            except exceptions.HttpNotFoundError:
+                LOGGER.exception('Certificate for program %s not configured', program_uuid)
```

Comment:
Log the username, too, in case an operator wants to know who should have received a certificate but didn't:
```
 LOGGER.exception('Certificate for program %s not configured, unable to award certificate to %s', program_uuid, username)
```